elements/map/helpers.ts

```diff
@@ -237,10 +238,15 @@ export async function registerProjectionFromCode(code: string | number) {
  */
```

Comment:
please add some JSDoc description, e.g. `@param extent the validity extent for the projection`